blinky-config/src/test/java/org/spideruci/analysis/config/definer/TestConfigFieldsDefiner.java

```diff
@@ -130,7 +130,7 @@ public void testConfigClassScanner() {
       
       assertEquals(isFieldExpectedToBeDefined, isFieldActuallyDefined);
       
-      if(isFieldExpectedToBeDefined ^ isFieldExpectedToBeDefined) { // different
+      if(isFieldExpectedToBeDefined ^ isFieldActuallyDefined) { // different
```

Comment:
Good catch. Since we are here, do you think we need this if-statement at all?

Reply:
I took a look at this again, at the current state I don't think it makes sense. After the assertEquals the test runner will stop the program, causing the program to not output the print statements. I think we should move the assertEquals below the if-else statement.